wrapper:
  config_file_name: utbm.yaml
  lidar_topic: /velodyne_points
  imu_topic: /imu/data
  lidar_type: 1 # velodyne
  time_unit: 1e-6
front_end:
  extrin_r: [1,0,0,
            0,1,0,
            0,0,1]
  extrin_t: [-0.5, 1.4, 1.5]
  use_inv: 0
ieskf:
  cov_gyroscope: 0.1
  cov_acceleration: 0.1
  cov_bias_acceleration: 0.0001
  cov_bias_gyroscope: 0.0001
